src/leakybucket.h

```diff
@@ -12,10 +12,15 @@
 #include <boost/chrono/chrono.hpp>
 #include <limits>
 
-extern const int64_t DEFAULT_MAX_RECV_BURST;
-extern const int64_t DEFAULT_AVE_RECV;
-extern const int64_t DEFAULT_MAX_SEND_BURST;
-extern const int64_t DEFAULT_AVE_SEND;
+// Variables for traffic shaping
+/** Default value for the maximum amount of data that can be received in a burst */
+static const int64_t DEFAULT_MAX_RECV_BURST = std::numeric_limits<long long>::max();
+/** Default value for the maximum amount of data that can be sent in a burst */
+static const int64_t DEFAULT_MAX_SEND_BURST = std::numeric_limits<long long>::max();
+/** Default value for the average amount of data received per second */
+static const int64_t DEFAULT_AVE_RECV = std::numeric_limits<long long>::max();
```

Comment:
Types should be consistent: int64_t vs. long long. I also think we should avoid compiler dependent int sizes in general unless it is a simple <32bit counter var.